src/modules/Channel/context/hooks/useResendMessageCallback.js

```diff
@@ -11,7 +11,7 @@ function useResendMessageCallback({
   return useCallback((failedMessage) => {
     logger.info('Channel: Resending message has started', failedMessage);
     const { messageType, file } = failedMessage;
-    if (failedMessage?.isResendable) {
+    if (typeof failedMessage?.isResendable === 'boolean') {
```

Comment:
```suggestion
    if (typeof failedMessage?.isResendable === 'boolean' && failedMessage.isResendable) {
```
?

Reply:
Ah yeah, actually the original logic has no problem. Probably I don't even need the fix in this file at least. So I'm checking with Uday again.